var/spack/repos/builtin/packages/resolve/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Resolve(CMakePackage, CudaPackage):
+    """FIXME: Put a proper description of your package here."""
+
+    homepage = "https://code.ornl.gov/ecpcitest/exasgd/resolve"
+    url = "https://code.ornl.gov/ecpcitest/exasgd/resolve.git"
+    maintainers("cameronrutherford", "pelesh", "ryandanehy")
+
+    # version("0.1.0", submodules=False, branch=develop)
+    # version("develop", submodules=False, branch=develop)
```

Comment:
I suggest we have the first release versioned 1.0. @kswirydo, what do you think?
```suggestion
    # version("1.0.0", submodules=False, branch=develop)
    # version("develop", submodules=False, branch=develop)
```